### PR TITLE
[Fix/fe 003.3.1] Final fixes for Release Candidate

### PR DESCRIPTION
**Pull request by bbogomilov1**

### CODE REVIEW

File: frontend/src/components/MainNavigation.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function MainNavigation() {
               Активни кампании
             </Link>
 
-            {isAdmin && (
+            {isAdmin && userPin === "1111111111" && (
```
Comment (ivan-parnarev) [inline frontend/src/components/MainNavigation.js:24]: Isn't isAdmin enough? why userPin also?

File: frontend/src/components/census/CensusModalComponent.tsx [hunk 2]
```diff
@@ -18,20 +18,39 @@ function CensusModalComponent({
...
   const [userData, setUserData] = useState<UserData>({
     userPin,
     campaignId: censusId,
     censusAnswers: [],
   });
 
+  const fetchCampaignDetails = async () => {
+    try {
+      const response = await axios.get(
+        `${API_URLS.ACTIVE_CAMPAIGNS}/${censusId}/CENSUS`
```
Comment (ivan-parnarev) [inline frontend/src/components/census/CensusModalComponent.tsx]: census with small letters?

File: frontend/src/components/voting/VotingActiveCampaignComponent.tsx [hunk 3]
```diff
@@ -1,16 +1,33 @@
...
   electionId,
-  electionCandidates,
 }: VotingActiveCampaignProps) {
   const [modalShow, setModalShow] = useState(false);
+  const [campaignDetails, setCampaignDetails] = useState<any>([]);
+
+  const fetchCampaignDetails = async () => {
+    try {
+      const response = await axios.get(
+        `${API_URLS.ACTIVE_CAMPAIGNS}/${electionId}/VOTING`
```
Comment (ivan-parnarev) [inline frontend/src/components/voting/VotingActiveCampaignComponent.tsx]: same with voting, small letters?

File: frontend/src/components/census/CensusModalComponent.tsx (hunk 2 above)
Comment (bbogomilov1) [reply frontend/src/components/census/CensusModalComponent.tsx]: We'll comment on that with the backend and decide for the next PR

**Pull request merged**